src/main/scala/akka/persistence/cassandra/journal/CassandraJournal.scala

```diff
@@ -245,13 +258,6 @@ class CassandraJournal(cfg: Config) extends AsyncWriteJournal with CassandraReco
         bs.setString("event_manifest", m.eventManifest)
         bs.setBytes("event", m.serialized)
 
-        if (session.protocolVersion.compareTo(ProtocolVersion.V4) < 0) {
-          bs.setToNull("message")
-          (1 to maxTagsPerEvent).foreach(tagId => bs.setToNull("tag" + tagId))
-        } else {
-          bs.unset("message")
-        }
```

Comment:
wouldn't it be enough with just two different prepared statements because for old versions there will not be any tags anyway (there is no support for materialized view)?